getting-started/sandbox-quick-start-guide.md

```diff
@@ -0,0 +1,439 @@
+
+###### 2.4.3.3 Sample Response
+|Code             |Response
+|-----------------|-----------------------------
+|200              |
+```
+{
+    "projectId": "abcd",
+    "transactionid": "3cfcb009-954d-4cbc-b941-855f04e87db6",
+    "filesCount": 2,
+    "fileDetails": [
+        {
+            "fileName": "test1.txt",
+            "fileStatus": "FileDownloaded",
+            "transactionTimestamp": "2021-02-02 10:45:02",
+            "fileTimestamp": "2021-02-02 17:42:24"
+        },
+        {
+            "fileName": "test2.pdf",
+            "fileStatus": "FileDownloaded",
+            "transactionTimestamp": "2021-02-02 10:45:02",
+            "fileTimestamp": "2021-02-02 17:42:24"
+        }
+    ]
+}
+```
+|4xx              |{"message":"error message"}
+
+#### 2.5 Commit Transaction
+After checking that the file has uploaded successfully, use the **\(Scan Transaction API\)** to commit the file for scan and transfer.
+
+![Display Step5](./images/Sandbox_filescan.jpg)
+
+##### 2.5.1 Resource Information
+|Method      |POST
+|------------|--------------------------------------------------------------------
+|URL         |https://api-sandbox.gdscft.govtechstack.sg/sandbox/v1/cft/transactions/{{sessionid}}/scan
+|What it does|Commits the transaction for scan and transfer
+
+##### 2.5.2 Request
+###### 2.5.2.1 Headers
+|Name                |Type     |Purpose                                  
+|------------        |---------|------------                             
+|x-api-key           |String   |API Key assigned to an individual project
+|authorization_token |String   |Authorization token
+|x-apigw-api-id      |String   |API Gateway Id
+
+
+##### 2.5.3 Response
+###### 2.5.3.1 JSON schema
+|Name           |Type                             
+|------------   |---------                        
+|projectId      |String                           
+|transactionId  |String                           
+|filesCount     |Integer                          
+|fileDetails    |JSON array of fileDetails schema 
+
+###### 2.5.3.2 fileDetails JSON schema
+|Name                 |Type        
+|------------         |---------    
+|fileName             |String       
+|fileStatus           |String       
+
+###### 2.5.3.3 Sample Response
+|Code             |Response
+|-----------------|-----------------------------
+|200              |
+```
+{
+    "projectId": "abcd",
+    "transactionid": "47e1da94-6c26-432f-9585-9f8927a71892",
+    "filesCount": 2,
+    "fileDetails": [
+        {
+            "fileName": "test1.txt",
+            "fileStatus": "FileSubmittedForScan"
+        },
+        {
+            "fileName": "test2.pdf",
+            "fileStatus": "FileSubmittedForScan"
+        }
+    ]
+}
+```
+|4xx              |
+
+
+### 3. Notification
+After the scan and transfer is complete, files will be available for download. CFT system will send a notification to the receiver via Webhook.*
+
+*(Webhook needs to be configured by receiver)
```

Comment:
appears simply as "*(Webhook needs to be configured by receiver)" in the browser